Dockerfile

```diff
@@ -0,0 +1,42 @@
+FROM ubuntu:20.04 as prebuilder
+
+ENV DEBIAN_FRONTEND noninteractive
+
+RUN apt-get update && apt-get install -y \
+    git \
+    imagemagick \
+    nodejs \
+    npm \
+    tesseract-ocr-deu \
+    && rm -rf /var/lib/apt/lists/*
+
+# allow ImageMagick to process PDFs
+RUN sed -i '$i\ \ <policy domain="coder" rights="read | write" pattern="PDF" />' /etc/ImageMagick-6/policy.xml
+
+FROM prebuilder as build-console
+
+COPY ./console /usr/local/hub/console
+WORKDIR /usr/local/hub/console
+RUN npm install --no-audit && npm run build
+
+FROM prebuilder as build-server
+
+COPY ./server /home/node/app
+WORKDIR /home/node/app
+
+RUN npm install --no-audit && npm run compile
+
+FROM prebuilder
```

Comment:
As we are not using the `node` base image here, we have to create the non-privileged user ourselves.
See https://github.com/nodejs/docker-node/blob/0544a5176c2a92c58c0809f5082e2877d796d394/14/buster/Dockerfile#L3-L4

Reply:
done